simple/imm_data.c

```diff
@@ -105,20 +121,20 @@ static int alloc_ep_res(struct fi_info *fi)
 	cq_attr.size = rx_depth;
 	ret = fi_cq_open(dom, &cq_attr, &rcq, NULL);
 	if (ret) {
-		printf("fi_cq_open() send comp %s\n", fi_strerror(-ret));
+		FI_PRINTERR("fi_cq_open: recv completion", ret);
```

Comment:
@shefty, there few more FI_* macros in shared.h. I guess it'll be better to create a new PR for that. I will only change then definition of FI_PRINTERR to have the line number in this PR.